pages/common/promtool.md

```diff
@@ -0,0 +1,20 @@
+# promtool
+
+> Tooling for the Prometheus monitoring system.
+> More information: <https://prometheus.io/docs/prometheus/latest/getting_started/>.
+
+- Check if the config files are valid or not (if present report errors):
+
+`promtool check config {{config_file}}.yml`
```

Comment:
I think it would be better to include the extension in the token syntax, because I think technically the user can pass a non-`.yaml` file&mdash;am I correct about this?
```suggestion
`promtool check config {{config_file.yml}}`
```